impl/src/main/resources/tck-tests.xml

```diff
@@ -52,6 +52,13 @@
                 </methods>
             </class>
 
+            <!-- CDITCK-622 -->
+            <class name="org.jboss.cdi.tck.tests.context.request.event.timeout.RequestScopeEventTimeoutTest">
```

Comment:
This should be `EJBJarDeploymentTest`, right?

Reply:
Yep, it is, I noticed it the second I sent the PR.
Good to know you're keeping an eye on it though ;-)